spec/requests/bookmarks_controller_spec.rb

```diff
@@ -6,9 +6,9 @@
   let(:bookmark_topic) { Fabricate(:topic) }
   let(:bookmark_user) { current_user }
 
-  before { sign_in(current_user) }
-
   describe "#create" do
+    before { sign_in(current_user) }
```

Comment:
I had to move this inside the describe blocks to add tests for signed out users.